x-pack/legacy/plugins/apm/server/routes/create_route.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as t from 'io-ts';
+import {
+  RouteFactoryFunction,
+  HttpMethod,
+  RouteParams,
+  RouteRequestHandler
+} from './typings';
+
+export function createRoute<
+  T extends string,
+  U extends HttpMethod,
+  V extends RouteParams,
+  W,
+  X extends RouteRequestHandler<V, W>
+>(cb: RouteFactoryFunction<T, U, V, W>) {
+  return cb;
+}
+
+createRoute(core => ({
+  path: '/foo',
```

Comment:
I assume this is just for testing purposes?

Reply:
Yep, will remove.